File: src/main/java/org/shark/example/controller/Controller.java
package org.shark.example.controller;

import org.shark.example.service.base.pojo.ResponseDto;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class Controller {

    @GetMapping
    public ResponseDto<Void> get() {
        return ResponseDto.<Void>builder().status(true).build();
    }
}

File: src/main/kotlin/org/shark/example/controller/Controller.kt
package org.shark.example.controller

import org.shark.example.service.base.pojo.ResponseDto
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class Controller {

    @GetMapping
    fun get(): ResponseDto<Void> {
        return ResponseDto.builder<Void>().status(true).build()
    }
}
